requirements.txt

```diff
@@ -1,7 +1,54 @@
-numpy>=1.14.3
-pandas>=0.22
-h5py>=2.8
-pytorch>=1.2
-tqdm>=4.26
-seaborn>=0.9
-matplotlib>=3.2
\ No newline at end of file
+appnope==0.1.2
```

Comment:
how did you create this dependency list and is created using on a Linux distro or what kind of bash / pip / ?

Reply:
Just a pip freeze from pyenv virtual environment. I don't know where appnope is from. Probably from another dependency.  Still, my virtual env is quite old. I might have some items not used.

Reply:
I feel like we should keep the old requirements just for simplicity